web/docs/platform-cli.md

```diff
@@ -18,150 +18,181 @@ pip install tenzir-platform
+tenzir-platform node delete <node_id>
+tenzir-platform node run [--name <node_name>] [--image <container_image>]
+```
 
-  We recommend using organization role rules to check if a user has a specific
-  role with an organization.
+### Description
 
-- **User Rule**: `tenzir-platform admin add-auth-rule user` allows access if the
-  `id_token` contains a field `sub` that exactly matches the provided
-  `<user_id>`.
+The following commands interact with the selected workspace. See [Authentication](#authentication)
+above for how to select a workspace:
+- `tenzir-platform node list` lists all nodes in the selected workspace,
+  including their ID, name, and connection status.
+- `tenzir-platform node ping` pings the specified node.
+- `tenzir-platform node create` registers a new node at the platform so that it
+  can be connected to the platform. Note that this neither starts a new node nor
+  configures one, it just creates a new API key that a node can use to connect
+  to the platform with.
+- `tenzir-platform node delete` removes a node from the platform. Note that this
+  does not stop the node, it just removes it from the platform.
+- `tenzir-platform node run` creates and registers an ad-hoc node, and starts it
+  on the local host. Requires Docker Compose to be available.
+  The node is temporary and will be deleted when the `run` command is stopped.
 
-## Manage Workspaces
+#### `<node_id>`
 
-### Synopsis
+The unique ID of the node, as shown in `tenzir-platform node list`.
+
+#### `<node_name>`
+
+The name of the node as shown in the app.
+
+#### `<container_image>`
+
+The Docker image to use for ad-hoc created node. We recommend using one of the
+following images:
+- `tenzir/tenzir:v4.11.2` to use the specified release.
+- `tenzir/tenzir:latest` to use the last release.
+- `tenzir/tenzir:main` to use the currnet development version.
+
+## Manage the Platform
+
+:::warning On-Premise Setup Required
+This functionality of the CLI can only be used in combination
+with an on-premise platform deployment, which is available to users
+of the [Sovereign Edition](https://tenzir.com/pricing).
+:::
+
+These CLI commands are only available to platform administrators.
+The `TENZIR_PLATFORM_OIDC_ADMIN_RULES` variable described
+[here](setup-guides/deploy-the-platform#identity-provider-idp) is used
+to define who is an administrator in your platform deployment.
+
+### Manage Workspaces
+
+#### Synopsis
 
 ```
 tenzir-platform admin list-global-workspaces
 tenzir-platform admin create-workspace <owner_namespace> <owner_id> [--name <workspace_name>]
 tenzir-platform admin delete-workspace <workspace_id>
 ```
 
-### Description
+#### Description
 
 The `tenzir-platform workspace admin list-global-workspaces`, `tenzir-platform
 admin create-workspace`, and `tenzir-platform admin delete-workspace` commands
 list, create, or delete workspaces, respectively.
 
-#### `<owner_namespace>`
+##### `<owner_namespace>`
```

Comment:
This looks really off compared to our operator docs when rendered, where we use h3 for all parameter descriptions. Do we need the extra nesting level?

Reply:
Not necessarily, I split it into two separate sections now.